src/main/java/com/puppycrawl/tools/checkstyle/filters/LoadPatchFileUtils.java

```diff
@@ -35,18 +35,18 @@
     /**
      * New line strategy that suppress all violations except from new lines.
      */
-    public static final String NEW_LINE = "newline";
+    public static final Strategy NEW_LINE = Strategy.NEWLINE;
 
     /**
      * Patched line strategy that suppress all violations except from new/changed lines.
      */
-    public static final String PATCHED_LINE = "patchedline";
+    public static final Strategy PATCHED_LINE = Strategy.PATCHEDLINE;
 
     /**
      * Context line strategy that suppress all violations except from new/changed lines
      * or matching context strategy.
      */
-    public static final String CONTEXT = "context";
+    public static final Strategy CONTEXT = Strategy.CONTEXT;
```

Comment:
you don't need these constants anymore, since there are enum values

Reply:
done